### ISSUE TITLE
Element.attributes should be a NamedNodeMap

### ISSUE THREAD

**Issue by paton** [tmpvar/jsdom#853]
``` 
document.createElement('div').attributes.toString();
```

Chrome:  "[object NamedNodeMap]"
jsdom:  "[object Object]"

Also...

```
var div = _window.document.createElement('div');
var attrs = [];

for (var prop in div.attributes) {
  if (div.hasOwnProperty(prop)) attrs.push(prop);
}
console.log(attrs)
```

Chrome: []
jsdom: [ '_ownerDocument', '_parentNode', '_readonly' ]

----

This is breaking a script that copies attributes from one element onto another.

Is this something that's fixable?

**Comment by Joris-van-der-Wel** [comment]
It is a `NamedNodeMap` now, however toString() will still return "[object Object]". 
The toString behaviour is defined in [Web IDL](http://www.w3.org/TR/WebIDL/#dfn-class-string). The toString is incorrect for a lot of objects in jsdom.

**Comment by domenic** [comment]
Oh hey, fixed this in 5.4.0.

**Comment by Frankusky** [comment]
I wasn't able to get NamedNodeMap, even using Object.prototype.toString.call(domElement.attribute) neither using domElement.attribute.toString(), both returns [object Object]

Im using the following code:

`let dom = new JSDOM("<!DOCTYPE html><div class="test"><script src="http://test.com/file.js" my-attribute="testing"></script>");
global.window = dom.window;
   global.document = dom.window.document;
      console.log(document.getElementsByTagName('script')[0].attributes.toString());
      console.log(Object.prototype.toString.call(document.getElementsByTagName('script')[0].attributes));
`
